v0/destinations/zendesk/transform.js

```diff
@@ -150,7 +150,10 @@ async function getUserId(message, headers, type) {
     type === "group"
       ? get(message, "context.traits")
       : getFieldValueFromMessage(message, "traits");
-  const userEmail = traits.email;
+  const userEmail = traits?.email;
+  if (!userEmail) {
+    throw new CustomError("Email is required for group call", 400);
```

Comment:
```suggestion
    throw new CustomError("Email ID is required for "group" events", 400);
```

Reply:
Also, can you double-check that this flow will always be hit for all the group events?

Reply:
We need the email for getting zendeskId which is needed for both the identify and group events.